Day3/day3.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python3
+
+
+def step1(bin_list):    
+    gamma = ''
+    epsilon = ''
+
+    for bit in range(len(bin_list[0])):
+        sum_of_0 = sum(number[bit] == '0' for number in bin_list)
+        sum_of_1 = sum(number[bit] == '1' for number in bin_list)
+
+        gamma += '0' if (sum_of_0 > sum_of_1) else '1'
+        epsilon += '1' if (sum_of_0 > sum_of_1) else '0'
```

Comment:
Python style tends to avoid parentheses wherever possible
These parentheses are notrequired and can be remove

Reply:
Done